src/skill/setNextAvailMachineTimerIntent.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+from utils import *
+from boto3.dynamodb.conditions import Key, Attr
+
+"""Dynamo resource and table"""
+dynamodb = boto3.resource('dynamodb', region_name='us-east-2')
+table = dynamodb.Table('Users')
+
+def setNextAvailMachineTimer(intent, session):
+
+    cardTitle = intent.get('name')
+    sessionAttributes = {}
+    shouldEndSession = False
+    
+    """Get buildingId from User table and machineType from session
+       Get available machines using API
+       If there are available machines, tell user how many
+       If not, set a timer for the next machine to become available
+    """
+    if session.get('user'):
+        userId = session.get('user').get('userId')
+        response = table.get_item(Key = {'userId': userId})
+        
+        if response.get('Item'):
+            buildingId = response.get('Item').get('buildingId')
+            machineType = intent.get('slots').get('machineType').get('value')
+            
+            """Allow for user to use plurals when interacting with Alexa"""
+            if machineType=='washers': 
+                machineType = 'washer'
+            elif machineType == 'dryers':
+                machineType = 'dryer'
+            
+            """Call API /buidlings/buildingId/machines?machineType=machineType&status=available"""
+            machineInfo = callApi(
+                url = 'https://go3ba09va5.execute-api.us-east-2.amazonaws.com/Test/buildings/{}/machines?machineType={}'
+                .format(buildingId, machineType)
+            )
+            if machineInfo != {'error': 'Building or machines not found'}:
+                numAvailMachines = len(machineInfo)
+                speechOutput = 'There are {} available {}s in your building right now'.format(str(numAvailMachines), machineType)
+                repromptText = 'There are {} available {}s in your building right now'.format(str(numAvailMachines), machineType)
+            else:

```

Comment:
The logic here seems a little off to me. What this code is saying to me is the following:

```
Get a list of the machines in building `buildingId` which are `machineType`
If we get a non-error response, tell the users how many machines we got from the response
If we get an error, make the same call again and find the lowest amount of time left, "set a timer" for that amount of time.
```

Might be a little nicer to do something like the following:

```
Get a list of the machines in building `buildingId` which are `machineType`
If we get a non error response, find the lowest amount of time left. 
    If it's zero, tell the user there are machines available
    Otherwise set a timer
If we get an error, tell the user that something went wrong. Ask them to set their building again (that's probably the issue)
```